test/app_test.dart

```diff
@@ -185,35 +186,60 @@ Future<void> main([List<String>? args]) async {
   });
 
   baasTest('Realm.logger', (configuration) async {
-    final oldLogger = Realm.logger;
+    Realm.logger = Logger.detached(generateRandomString(10))..level = RealmLogLevel.all;
```

Comment:
Why not move these tests in `realm_logger_test.dart`

Reply:
Having logger test in different files is good because the tests are running in different isolates. Since these tests don't require to be marked as loggerTest `setupTests(args, loggerTest: true)` because they replace the logger themselves, it is not a problem they to be here.

Reply:
We shouldn't group the tests based on the implementation. We are grouping them based on feature. I don't see the reason to have logger tests all over the place. They should really be done in one file regardless if they are "logger" tests or not or the behavior they are testing. This makes it easy to maintain.